docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,350 @@
+    ```
+
+    :::tip
+
+    To enable CSI-assisted cloning into larger PVCs than the source PVC, set `allowVolumeExpansion : true` on your
+    defined Storage Class.
+
+    :::
+
+3.  When the CDI clones a PVC, it runs under the `default` service account in the namespace of the target PVC. When the
+    source PVC is in a different namespace, you must give the required permissions to the service account. The
+    [VMO pack](../vm-management.md) version 4.2.0 does this automatically through its default charts specification. This
+    configuration uses the `vmEnabledNamespaces` option to specify the namespaces for which the permissions are
+    configured.
+
+    ```yaml {8}
+    charts:
+      virtual-machine-orchestrator:
+        # This namespace will be used to store golden images
+        goldenImagesNamespace: "vmo-golden-images"
+
+        # These namespaces will be created and set up to deploy VMs into
+        vmEnabledNamespaces:
+          - "default"
+          - "virtual-machines"
+    ```
+
+4.  By default, the CDI contains an internal database of CSIs, with the CSI supported access modes, volume modes, and
+    CSI-assisted cloning support. However, we recommend creating a `StorageProfile` to adjust these parameters to your
+    desired values.
+
+    Create another manifest in this cluster profile and add the following code snippet.
+
+    This snippet has three notable configurations that define the storage class.
+
+    - The storage class has the name `ceph-block`.
+    - The storage class supports the `Block` volume mode with the `ReadWriteMany` access mode.
+    - The storage class uses `csi-clone` for cloning operations.
+
+    <br />
+
+    ```yaml {4,8,10,11}
+    apiVersion: cdi.kubevirt.io/v1beta1
+    kind: StorageProfile
+    metadata:
+      name: ceph-block
+    spec:
+      claimPropertySets:
+        - accessModes:
+            - ReadWriteMany
+          volumeMode: Block
+      cloneStrategy: csi-clone
+    ```
+
+    Expand the following section to review how to use storage profiles for CSI-assisted cloning.
+
+    <details>
+
+    <summary>Storage Profiles in VM Templates</summary>
+
+    If Storage Profiles are not manually defined, the CDI will automatically generate one for every Storage Class, based
+    on the information in its internal database. It is recommend to verify these parameters are accurate for your use
+    case and if not, overwrite them by defining the Storage Profile manually.
+
+    As noted in the
+    [CDI documentation](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/datavolumes.md#storage),
+    you can optionally replace the target `pvc` sections in your `vmTemplate` resources with a `storage` section once
+    you have defined a storage profile.
+
+    Using storage instead of PVC will automatically discover default values for `accessMode`, `volumeMode` and
+    optionally even the size from the source PVC and its associated storage profile.
```

Comment:
```suggestion
    Using storage instead of PVC will automatically discover default values for `accessMode`, `volumeMode` and,
    optionally, even the size from the source PVC and its associated storage profile.
```